sdks/python/apache_beam/io/filesystem.py

```diff
@@ -668,12 +668,12 @@ def exists(self, path):
 
   @abc.abstractmethod
   def size(self, path):
-    """Get size of path on the FileSystem.
+    """Get size in bytes of a file on the FileSystem.
 
     Args:
-      path: string path in question.
+      path: string filepath of file.
```

Comment:
Could I ask you to add this to the class comment (I can't comment there):
Note to implementors: "path" arguments will be URLs in the form scheme://foo/bar. The exception is LocalFileSystem, which gets unix-style paths in the form /foo/bar.